schemas/iso19139/src/main/plugin/iso19139/layout/utility-tpl-multilingual.xsl

```diff
@@ -38,8 +38,17 @@
         <xsl:value-of select="xslutil:getLanguage()" />
       </xsl:when>
       <xsl:otherwise>
-        <xsl:value-of select="$metadata/gmd:language/gco:CharacterString|
-       $metadata/gmd:language/gmd:LanguageCode/@codeListValue"/>
+          <xsl:choose>
```

Comment:
Can't be changed to use `langId_from_gmdlanguage19139`?

Reply:
It probably can however I would need to add an xsl:include  and I was not sure if that was going to cause any conflicts - I can try it and see if it causes any issues.